src/se/simple_type.rs

```diff
@@ -0,0 +1,396 @@
+//! Contains Serde `Serializer` for XML [simple types] [as defined] in the XML Schema.
+//!
+//! [simple types]: https://www.w3schools.com/xml/el_simpletype.asp
+//! [as defined]: https://www.w3.org/TR/xmlschema11-1/#Simple_Type_Definition
```

Comment:
If any of this can sensibly be shared, it might make sense in `crate::escapei`

Reply:
This is hard to say now which of them would be useful, can be re-exported later in follow-up PRs.